projects/1.20/assets/tofucraftreload/tofucraft/lang/zh_cn.json

```
@@ -0,0 +1,626 @@
+  "entity.tofucraft.shudofuspider" : "臭豆腐蜘蛛",
+  "entity.tofucraft.tofu_boat" : "豆腐船",
+  "entity.tofucraft.tofu_chest_boat" : "豆腐运输船",
+  "advancements.tofucraft.root.title" : "豆腐工艺",
+  "advancements.tofucraft.root.description" : "一边制做豆腐，一边发现新的可能吧！",
+  "advancements.tofucraft.make_saltpan.title" : "搭建盐田",
+  "advancements.tofucraft.make_saltpan.description" : "合成盐田",
+  "advancements.tofucraft.bittern_bottle.title" : "获得盐卤",
+  "advancements.tofucraft.bittern_bottle.description" : "向盐田中加水后，用瓶子从盐田中获得盐卤。",
+  "advancements.tofucraft.bitter_better.title" : "一点都不觉得苦",
+  "advancements.tofucraft.bitter_better.description" : "为了获得盐和盐卤，制作盐炉吧",
+  "advancements.tofucraft.make_tofu.title" : "豆腐!",
+  "advancements.tofucraft.make_tofu.description" : "使用盐卤让豆浆凝固，从而获得豆腐，试着破坏它吧",
+  "advancements.tofucraft.harder_tofu.title": "更硬了",
+  "advancements.tofucraft.harder_tofu.description": "通过在上下放置石头，来压制豆腐",
+  "advancements.tofucraft.tofustick.title": "豆腐世界通行证",
+  "advancements.tofucraft.tofustick.description": "取得豆腐魔杖",
+  "advancements.tofucraft.tofuworld.title": "豆腐世界！",
+  "advancements.tofucraft.tofuworld.description": "进入豆腐世界",
+  "advancements.tofucraft.tofugem.title": "能量的来源",
+  "advancements.tofucraft.tofugem.description": "获得豆腐宝石",
+  "advancements.tofucraft.tofudiamond.title": "豆腐世界的钻石？",
+  "advancements.tofucraft.tofudiamond.description": "获得9个钻石豆腐粒，制作钻石豆腐",
+  "advancements.tofucraft.tofuvillage.title": "可爱的豆腐之家",
+  "advancements.tofucraft.tofuvillage.description": "找到豆腐村庄",
+  "advancements.tofucraft.trade_with_tofunian.title": "新鲜交易",
+  "advancements.tofucraft.trade_with_tofunian.description": "和可爱的豆腐村民进行交易",
+  "advancements.tofucraft.nottofu.title": "像豆腐但不完全是豆腐",
+  "advancements.tofucraft.nottofu.description": "找到豆腐茎杆",
+  "advancements.tofucraft.tofufarmers_tool.title": "豆腐农家的秘传工具",
+  "advancements.tofucraft.tofufarmers_tool.description": "豆腐村民的锄头",
+  "advancements.tofucraft.tofu_industrial.title": "豆腐工业时代",
+  "advancements.tofucraft.tofu_industrial.description": "制作豆腐能存储器",
+  "advancements.tofucraft.nether_soybeans.title": "地狱的大豆",
+  "advancements.tofucraft.nether_soybeans.description": "找到地狱大豆",
+  "advancements.tofucraft.nether_tofu.title": "来自地狱的小点心",
+  "advancements.tofucraft.nether_tofu.description": "制作地狱豆腐",
+  "advancements.tofucraft.very_strange_soybeans.title": "非常奇怪的大豆",
+  "advancements.tofucraft.very_strange_soybeans.description": "找到灵魂大豆",
+  "advancements.tofucraft.tofu_mental.title": "有情感的豆腐",
+  "advancements.tofucraft.tofu_mental.description": "使用哭泣黑曜石压制豆腐",
+  "advancements.tofucraft.fermentation_basics.title": "发酵基础",
+  "advancements.tofucraft.fermentation_basics.description": "制作曲种，等待发酵",
+  "advancements.tofucraft.miso.title": "一个大豆;两种制品",
+  "advancements.tofucraft.miso.description": "和豆腐一样，通过在木桶上下放置石头，来制作味噌和酱油",
+  "advancements.tofucraft.sniffed_revenge.title": "嗅觉的复仇",
+  "advancements.tofucraft.sniffed_revenge.description": "为了报复嗅觉灵敏的监守者,把辣椒扔向它吧！",

```

```suggestion
  "advancements.tofucraft.sniffed_revenge.description": "为了报复嗅觉灵敏的监守者，把辣椒扔向它吧！",
```
我们遵循《标点符号规范》，需要全文使用中文全角符号。
https://cfpa.site/TransRules/
此类问题请全文通查